content/en/service_management/service_level_objectives/_index.md

```diff
@@ -51,18 +51,31 @@ Service Level Agreement (SLA)
 Error Budget
 : The allowed amount of unreliability derived from an SLO's target percentage (100% - target percentage) that is meant to be invested into product development.
 
+## SLO Types
+
+When creating SLOs, you can choose from the following types:
+
+**Metric-based SLOs**: can be used for count-based data streams, the SLI is based on the sum of good events divided by the sum of total events.
+
+**Time Slice SLOs or Monitor-based SLOs**: can be be used for time-based data sets, the SLI is based on the amount of time your system exhibits good behavior divided by the total time.
+  - Time Slice SLOs: do not require a Datadog monitor
+  - Monitor-based SLOs: must be based on a new or existing Datadog monitor
+
+For a summary of the key differences between the available SLO types, see the [comparison chart][1].
+
+
 ## Setup
 
-Use Datadog's [Service Level Objectives status page][1] to create new SLOs or to view and manage all your existing SLOs. You can also add [SLO widgets](#slo-widgets) to your dashboards to visualize your SLO statuses at a glance.
+Use Datadog's [Service Level Objectives status page][2] to create new SLOs or to view and manage all your existing SLOs. 
 
 ### Configuration
 
-1. On the [SLO status page][1], select **New SLO +**.
-2. Define the source for your SLO. You can create an SLO from [metrics][2] or [monitors][3].
+1. On the [SLO status page][2], select **New SLO +**.
+2. Define the source for your SLO. You can create an SLO from [metrics][3] or [monitors][4].
```

Comment:
Should also mention time slice SLOs as an option. We should say, "You can create an SLO with any of the following types: Metric-based, Monitor-based, or Time Slices.